ocaml/xapi/xapi_guest_agent.ml

```diff
@@ -218,6 +223,10 @@ let get_initial_guest_metrics (lookup: string -> string option) (list: string ->
     | Some value -> ["data-ts",value]
     | None -> [] in
 
+  let cant_suspend = match lookup "data/cant_suspend_reason" with
```

Comment:
Where is `cant_suspend` used?

Reply:
I'll need to refactor this a bit, I ended up with some code duplication.